.drone.yml

```diff
@@ -28,6 +30,25 @@ steps:
     when:
       event:
         - push
+  # this step will run on every push, and also via cron every day
+  - name: mirror-images-daily
+    image: rancher/dapper:v0.5.7
+    volumes:
+      - name: docker
+        path: /var/run/docker.sock
+    environment:
+      DOCKER_PASSWORD:
+        from_secret: docker_password
+      DOCKER_USERNAME:
+        from_secret: docker_username
+    commands:
+      - cat images-list-daily | dapper
+    settings:
+      custom_dns: 1.1.1.1
+    when:
+      event:
+      - push
+      - cron
```

Comment:
When does this run exactly? I don't see a default or a configuration when it needs to run, but I'm also not familiar with Drone and cron triggers.

Reply:
@paraglade setup a cron in drone that runs every day, that will trigger an event here.